picard/ui/item.py

```diff
@@ -100,27 +100,38 @@ def error_append(self, msg):
     def clear_errors(self):
         self._errors = []
 
-    def _cover_art_description(self, images, detailed=True):
-        """Return the number of cover art images in `images` for display in the UI
+    @property
+    def _images(self):
+        return self.metadata.images
```

Comment:
When I suggested passing `self.metadata.images`  to the helper methods I had two issues in mind:

1. I wasn't sure if we should rely upon `Item`  having a `metadata` attribute. Of course classes using those helper methods currently always would have

2. Wanted to keep it flexible so it can also be used for other metadata objects e.g. (self.orig_metadata)

Of course code otherwise becomes a bit nicer this way. Thoughts?

Reply:
I thought about it too, this is why I introduced `_images`:
- we could check for `metadata.images` attributes there
- or alternatively using `orig_images`

But, for now, I see only one case we may want to do that, when there's no images in metadata but some in orig_metadata, like we do in https://github.com/metabrainz/picard/blob/de45f3cd9d0a5ba5953d47d5c231e37c83a8bf63/picard/ui/infodialog.py#L125-L142